README.md

```diff
@@ -120,9 +120,9 @@ Example:
 
 ```js
 var fsBlobStore = require('fs-blob-store')  // an in-memory blob store
```

Comment:
actually this is not correct anymore, it needs to be `fs-pull-blob-store` and the comment is wrong as this is not an in-memory blob store, but an fs blob store